MapboxNavigation/FeedbackViewController.swift

```diff
@@ -3,29 +3,46 @@ import MapboxCoreNavigation
 import AVFoundation
 
 extension FeedbackViewController: UIViewControllerTransitioningDelegate {
-    func animationController(forDismissed dismissed: UIViewController) -> UIViewControllerAnimatedTransitioning? {
+    @objc public func animationController(forDismissed dismissed: UIViewController) -> UIViewControllerAnimatedTransitioning? {
         abortAutodismiss()
         return DismissAnimator()
     }
     
-    func animationController(forPresented presented: UIViewController, presenting: UIViewController, source: UIViewController) -> UIViewControllerAnimatedTransitioning? {
+    @objc public func animationController(forPresented presented: UIViewController, presenting: UIViewController, source: UIViewController) -> UIViewControllerAnimatedTransitioning? {
         return PresentAnimator()
     }
     
-    func interactionControllerForDismissal(using animator: UIViewControllerAnimatedTransitioning) -> UIViewControllerInteractiveTransitioning? {
+    @objc public func interactionControllerForDismissal(using animator: UIViewControllerAnimatedTransitioning) -> UIViewControllerInteractiveTransitioning? {
         return interactor.hasStarted ? interactor : nil
     }
 }
 
-typealias FeedbackSection = [FeedbackItem]
-
-class FeedbackViewController: UIViewController, DismissDraggable, UIGestureRecognizerDelegate {
+/**
+ The `FeedbackViewControllerDelegate` protocol provides methods for responding to feedback events.
+ */
+@objc public protocol FeedbackViewControllerDelegate {
+    
+    /**
+     Called when the user opens the feedback form.
+     */
+    @objc optional func  feedbackViewControllerDidOpen(_ feedbackViewController: FeedbackViewController)
     
-    typealias SendFeedbackHandler = (FeedbackItem) -> Void
+    /**
+     Called when the user submits a feedback event.
+     */
+    @objc optional func feedbackViewController(_ feedbackViewController: FeedbackViewController, didSend: FeedbackItem, UUID: UUID)
```

Comment:
* [ ] Distinguish between the argument label and argument name: `didSend feedbackItem: FeedbackItem`
* [ ] The Swift name is currently `feedbackViewController(_:didSend:UUID:)`; it should be `feedbackViewController(_:didSend:uuid:)`
* [ ] The Objective-C name is currently `-feedbackViewController:didSend:UUID:`; it should be `-feedbackViewController:didSendFeedbackItem:UUID:`